tests/hooks/test_hive_hook.py

```diff
@@ -282,6 +283,13 @@ def test_get_max_partition_from_valid_part_specs(self):
     def test_get_metastore_client(self):
         self.assertIsInstance(self.hook.get_metastore_client(), HMSClient)
 
+    @mock.patch("airflow.hooks.hive_hooks.HiveMetastoreHook.get_connection",
+                **{'return_value': [Connection(host="localhost", port=9802)]})
```

Comment:
Why not `return_value=[Connection(…)]` instead of using the `**` syntax?

Reply:
Preferred syntax does not work properly in all cases supported by mock. I often use only one syntax, so that I do not have to think which is right at the moment.
Reference: 
https://github.com/apache/airflow/blob/49b98ab2b5dc37c69598d485d62dd39372ab6aa9/tests/contrib/hooks/test_gcp_natural_language_hook.py#L57-L68

Reply:
> Your scientists were so preoccupied with whether or not they could, they didn’t stop to think if they should.

The suggested syntax does not work, indeed, when you have invalid identifiers as keyword arguments. `return_value` is not one of them